<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraControlWidget</class>
 <widget class="QWidget" name="CameraControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>247</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3" rowspan="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QRadioButton" name="hardware_trigger_radioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Hardware</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">trigger_source_buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="software_trigger_radioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Software</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">trigger_source_buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="triggered_radioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Triggered</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">capture_mode_buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="continuous_radioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Continuous</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">capture_mode_buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="capture_mode_label">
          <property name="text">
           <string>Capture Mode</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="software_trigger_pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Software Trigger</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="trigger_source_label">
          <property name="text">
           <string>Trigger Source:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="camera_comboBox">
          <item>
           <property name="text">
            <string>Select Imaging System...</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="serial_label">
          <property name="text">
           <string>Serial Number: xxxxxxxx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="arm_pushButton">
          <property name="text">
           <string>Arm Camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Start Camera</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="exposure_label">
            <property name="text">
             <string>Exposure Time (ms)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="exposure_lineEdit">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="exposure_pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Update Exposure Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="trigger_source_buttonGroup"/>
  <buttongroup name="capture_mode_buttonGroup"/>
 </buttongroups>
</ui>
